Core/Core/Kits/Applications.cs

```diff
@@ -124,6 +125,11 @@ public static HostApplication GetHostAppFromString(string? appname)
       return Civil;
     }
 
+    if (appname.Contains("civil3d"))
```

Comment:
what is the difference with "civil"? because this will always return the previous one I believe because it will contain already "civil"

Reply:
Ah good point, will fix this